components/input/gpio/gpio_test.go

```diff
@@ -275,18 +273,17 @@ func TestGPIOInput(t *testing.T) {
 		})
 	})
 
-	// Test with debounce disabled
 	t.Run("inverted button press with debounce disabled", func(t *testing.T) {
 		s := setup(t)
 		defer teardown(t, s)
 
-		for i := 0; i < 20; i++ {
+		iterations := 50
+
+		for i := 0; i < iterations; i++ {
 			err := s.b.Digitals["interrupt2"].Tick(s.ctx, true, uint64(time.Now().UnixNano()))
 			test.That(t, err, test.ShouldBeNil)
-			time.Sleep(time.Millisecond)
```

Comment:
i'm not sure what value these sleeps served. my understanding for this test was that commands should be issued as fast as possible and ensure that none get debounced.

Reply:
Was just "best practice" to keep things from being CPU bound. E.g. depending on load and cpu power, this might go a thousand times faster or slower on different runs/machines, and the sleep means it'll be "approximately" the same anywhere. If it works with them removed, I'm fine with them gone though.